src/lib.rs

```diff
@@ -3,6 +3,7 @@ pub mod db;
 pub mod file_uploader;
 pub mod github;
 pub mod handlers;
+pub mod index;
```

Comment:
not sure why we need multiple levels of nesting - i.e. could this code not have gone in handlers?
Note: Im unfamiliar with the domain though

Reply:
It is not an API call handler, it is semantically doing something different. I am not sure why would it be better to have it non-isolated in its own module